deploy/stacks/lambda_api.py

```diff
@@ -223,73 +224,94 @@ def __init__(
                 )
             )
 
-        if custom_auth is not None:
-            # Create the custom authorizer lambda
-            custom_authorizer_assets = os.path.realpath(
-                os.path.join(
-                    os.path.dirname(__file__),
-                    '..',
-                    'custom_resources',
-                    'custom_authorizer',
-                )
+        # Create the custom authorizer lambda
+        custom_authorizer_assets = os.path.realpath(
+            os.path.join(
+                os.path.dirname(__file__),
+                '..',
+                'custom_resources',
+                'custom_authorizer',
             )
+        )
+        ## GET CONGITO USERL POOL ID and APP CLIENT ID
 
-            if not os.path.isdir(custom_authorizer_assets):
-                raise Exception(f'Custom Authorizer Folder not found at {custom_authorizer_assets}')
+        if not os.path.isdir(custom_authorizer_assets):
+            raise Exception(f'Custom Authorizer Folder not found at {custom_authorizer_assets}')
 
-            custom_lambda_env = {
-                'envname': envname,
-                'LOG_LEVEL': 'DEBUG',
-                'custom_auth_provider': custom_auth.get('provider'),
-                'custom_auth_url': custom_auth.get('url'),
-                'custom_auth_client': custom_auth.get('client_id'),
-                'custom_auth_jwks_url': custom_auth.get('jwks_url'),
```

Comment:
Are we removing the `custom_auth_jwks_url` URL completely ? and using the `.well-known/openid-configuration` instead ?

Reply:
correct